extras/ds3231/component.mk

```diff
@@ -0,0 +1,9 @@
+# Component makefile for extras/ds3231
+
+# expected anyone using bmp driver includes it as 'ds3231/ds3231.h'
```

Comment:
I took the bmp driver dir as skeleton, missed updating the comment. Will fix this in next version.